apps/asap-server/src/entities/user.ts

```diff
@@ -91,62 +65,98 @@ export const createSchema = Joi.object({
   connections: Joi.string(),
 });
 
-export const parseGraphQLUserTeamConnection = (
-  item: CMSGraphQLUserTeamConnection,
+export const parseGraphqlUserTeamConnection = (
```

Comment:
Keep it consistent across functions and variables
```suggestion
export const parseGraphQLUserTeamConnection = (
```

Reply:
Or go the other way around, always lowercase, but we should try to keep it consistent